cuda/test/matrix/csr_kernels.cpp

```diff
@@ -926,4 +938,75 @@ TEST_F(Csr, OutplaceAbsoluteComplexMatrixIsEquivalentToRef)
 }
 
 
+TEST_F(Csr, CalculateNnzPerRowIsEquivalentToRef)
```

Comment:
nit: this should point to submatrix computation somehow
```suggestion
TEST_F(Csr, CalculateSubmatrixNnzPerRowIsEquivalentToRef)
```